src/pages/home/report/ReportActionsList.js

```diff
@@ -176,6 +176,48 @@ function ReportActionsList({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [report.lastReadTime]);
 
+    // In the component we are subscribing to the arrival of new actions.
+    // As there is the possibility that there are multiple instances of a ReportScreen
+    // for the same report, we only ever want one subscription to be active, as
+    // the subscriptions could otherwise be conflicting.
+    const newActionUnsubscribeMap = {};
```

Comment:
Should we put this outside the function component? I see that previously, we've done that. I think it makes sense because it has nothing to do with the component to render or update their state.